### ISSUE TITLE
Make some "favorite settings" to short links in the context menu

### ISSUE THREAD

**Issue by lokimo** [bpierre/gtranslate#52]
Hi there,

this is only a feature request.
I often switch languages from "auto detect -> english" to "auto detect -> german" and back.
So if we would have short links in the context menu we would be able to save time going through all the languages list.

**Comment by mote0230** [comment]
Do you think it would be a good idea to translate to 2 languages at the same time?

**Comment by lokimo** [comment]
Oh, yes, that would also be very useful. I think even better in helping to learn a language.

Another solution would be to implement some kind of rules which take the result of the auto-detect language and then decide the target language.

**Comment by bpierre** [comment]
I agree, it would be nice to have a quick way to switch back to the previously selected languages (see #22 too).

> Do you think it would be a good idea to translate to 2 languages at the same time and display both results next to each other?

It looks like a specific use case to me, but maybe in the settings?
